WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift

```diff
@@ -394,10 +394,16 @@ extension ReaderFollowedSitesViewController: WPTableViewHandlerDelegate {
             return
         }
 
-        // Reset the site icon first to address: https://github.com/wordpress-mobile/WordPress-iOS/issues/8513
-        cell.imageView?.image = .siteIconPlaceholder
+        var placeholderImage: UIImage = .siteIconPlaceholder
+        if site.isP2Type {
+            placeholderImage = UIImage.gridicon(.p2, size: CGSize(width: 40, height: 40))
+            cell.imageView?.layer.borderColor = UIColor.divider.cgColor
```

Comment:
If it doesn't have a border, it doesn't need a border color, right?

Reply:
Also, if the border color can be removed, creating the `placeholderImage` can probably be reduced to something like:
`let placeholderImage = site.isP2Type ? UIImage.gridicon(.p2, size: CGSize(width: 40, height: 40)) : .siteIconPlaceholder`

Reply:
maybe I was saving it for later, who knows 😁 
deleted ✅